pkgs/development/python-modules/pyicumessageformat/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "pyicumessageformat";
+  version = "1.0.0";
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-s+l8DtEMKxA/DzpwGqZSlwDqCrZuDzsj3I5K7hgfyEA=";
+  };
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ pname ];
+
+  meta = with lib; {
+    description = "An unopinionated Python3 parser for ICU MessageFormat";
```

```suggestion
    description = "Unopinionated Python3 parser for ICU MessageFormat";
```
https://github.com/NixOS/nixpkgs/blob/3836c3482484025821c9a55ca3db45c204ea5646/pkgs/README.md?plain=1#L378

---

Done